app/backend/hiStoryBackend/api/migrations/0008_memorymedia.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.1.2 on 2018-10-26 18:40
+
+import api.models
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0007_memorypost'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='MemoryMedia',
+            fields=[
```

Comment:
Let's also add the order it appears in the memory post so that we have a more elegant fix for the same file problem?